PRIVACY.md

```diff
@@ -0,0 +1,14 @@
+Privacy Notice
```

Comment:
this should also not look like it's added: it was already in main! 
(Note that I've changed the base to main for this repo)